example_queries/breadth_histogram_rust.cql

```diff
@@ -0,0 +1,2 @@
+MATCH x-->y: m, WHERE x.service_name == "frontend-v1_plugin", GROUP x.breadth BY histogram,
```

Comment:
The queries should be the same, no? Or are these queries that only work in the simulator? Why do they only work in the simulator?

Reply:
The simulator automatically adds a "_plugin" to the name of any plugin.  When you ask the plugin what its ID is, it gives the answer with the "_plugin" attached.  So the queries are nearly the same, except every time we mention a node name, we add "_plugin".

Reply:
That seems like a design flaw. There should be no difference between the simulator and the kubernetes environment in terms of the queries. That kind of defeats the purpose if we have to customize.

Reply:
True.  I changed the simulator so that the _plugin part is no longer part of the plugin's name as far as the plugin knows.  This fixed it.